src/main/python/org/broadinstitute/hellbender/scorevariants/tests/test_dataset.py

```diff
@@ -0,0 +1,200 @@
+from unittest import TestCase
```

Comment:
@KevinCLydon There are a number of Python-based unit tests under `src/main/python/org/broadinstitute/hellbender/scorevariants/tests` -- ideally we should find a way to run these as well, perhaps in `NVScoreVariantsIntegrationTest` via a `PythonScriptExecutor`.